frontend/app/model/ResponsiveImage.scala

```diff
@@ -51,7 +51,10 @@ case class ResponsiveImageGroup(
 
   private val sortedImages = availableImages.sortBy(_.width)
 
-  val smallestImage = sortedImages.head.path
+  // We expect to have at least one availableImage. In the rare case we don't,
+  // set the path to about:blank, which is still a valid URL whose resource is an empty string.
+  // http://www.w3.org/TR/2011/WD-html5-20110525/fetching-resources.html#about:blank
+  val smallestImage = sortedImages.headOption.map(_.path).getOrElse("about:blank")
```

Comment:
Probably the better solution is to omit the `<img>` element altogether if we can't find a secure URL for it, rather than setting the `src` to `about:blank`. This will require more refactoring though so I've [put up a separate ticket for it](https://trello.com/c/S1Cq82tD/268-handle-cases-when-we-can-t-get-a-secure-path-to-an-image-from-capi)